backend/routes/index.js

```diff
@@ -139,9 +139,13 @@ router.patch("/:id", async (req, res) => {
 
     if (!todo) return res.status(404).send("Todo not found...");
 
-    const updatedTodo = await Todo.findByIdAndUpdate(req.params.id, {
-      isComplete: !todo.isComplete,
-    });
+    const updatedTodo = await Todo.findByIdAndUpdate(
+      req.params.id,
+      {
+        isComplete: !todo.isComplete,
```

Comment:
change this to false instead of negating the current state of isComplete. This way we understand easily that we changing the isComplete to false.